packages/babel-parser/test/helpers/runFixtureTests.js

```diff
@@ -7,11 +7,25 @@ export function runFixtureTests(fixturesPath, parseFunction) {
     fixtures[name].forEach(function(testSuite) {
       testSuite.tests.forEach(function(task) {
         const testFn = task.disabled ? it.skip : it;
+        const fs = require("fs");
```

Comment:
would prefer having those on top, mixing cjs/esm requires/imports is imho a bad practice in general (unless ofc rly necessary in some cases)

I think (although not 100% sure) most of babel packages are compiled now with CJS lazy mode, so moving it to top shouldnt even cost initialization time.